tests/common/snappi/common_helpers.py

```diff
@@ -100,15 +91,68 @@ def get_egress_lossless_buffer_size(host_ans):
     egress_lossless_pool = buffer_pools[profile_name]
     return int(egress_lossless_pool['size'])
 
+def get_lossless_buffer_size(host_ans):
+    """
+    Get egress lossless buffer size of a switch, unless an 8102 switch,
+    in which case, get the ingress lossless buffer size
+    Args:
+        host_ans: Ansible host instance of the device
+    Returns:
+        total switch buffer size in byte (int)
+    """
+    config_facts = host_ans.config_facts(host=host_ans.hostname,
+                                         source="running")['ansible_facts']
+    is_8102 = True if '8102' in host_ans.facts['platform'] else False
```

Comment:
is_8102 -> is_cisco_8102. We should see if both "cisco" and "8102" are in the host_ans.facts['platform']